app/assets/javascripts/save-editor-progress.es6

```diff
@@ -3,16 +3,33 @@
 
     function submitPracticeEditorSaveForm() {
         $('#practice-editor-save-button').on('click', () => {
-            if ($('#form')[0].checkValidity()) {
-                    $('#form').submit();
+            let $form = $('#form');
+            if ($form[0].checkValidity()) {
+                    $form.submit();
             } else {
-                $('#form')[0].reportValidity();
+                $form[0].reportValidity();
+            }
+        });
+    }
+
+    function saveEditorProgressOnContinue() {
+        $('.continue-and-save').on('click', (event) => {
+            event.preventDefault();
+            let $form = $('#form');
+
+            $form.append(`<input type='hidden' name='next' value=true>`);
+
+            if ($form[0].checkValidity()) {
+                $form.submit();
+            } else {
+                $form[0].reportValidity();
```

Comment:
```
if ($form[0].checkValidity()) {
    $form.submit();
} else {	           
     $('#form')[0].reportValidity();	                
}
```
this can be DRY
make a function and send in the `$form`
then use in both functions